scripts/CMakeLists.txt

```diff
@@ -52,9 +52,18 @@ if (INSTALL_SYSTEM_FILES)
 			DESTINATION ${BASH_COMPLETION_COMPLETIONSDIR}
 			RENAME kdb)
 
+	set (FISH_COMPLETION_COMPLETIONSDIR "/usr/share/fish/vendor_completions.d")
+	find_package (PkgConfig)
+	if (PKG_CONFIG_FOUND)
+		include (LibFindMacros)
+		pkg_get_variable (FISH_COMPLETION_COMPLETIONSDIR fish completionsdir)
```

Comment:
The problem should be fixed in commit [ef38dbd0](https://github.com/sanssecours/libelektra/commit/ef38dbd0e489039bec4bd20c3b8f411c70c90bfd).